docs/call.md

```diff
@@ -1,8 +1,9 @@
 # Call API
 
-* Base endpoint for API v1 is: `/ocs/v2.php/apps/spreed/api/v1`
-* Base endpoint for API v2 is: `/ocs/v2.php/apps/spreed/api/v2`
-* Base endpoint for API v3 is: `/ocs/v2.php/apps/spreed/api/v3`
+* API v1: 🏁 Removed with API v4
+* API v2: 🏁 Removed with API v4
+* API v3: 🏁 Removed with API v4
+* API v4: Base endpoint `/ocs/v2.php/apps/spreed/api/v4`
```

Comment:
So this means all clients will stop working with Talk 12 and an upgrade will be mandatory.
If there are no scenarios where someone would not want to upgrade their apps (branding?) I guess this is ok.
Also it won't be possible to first upgrade all client apps first and then the server, so the migration will have to be done "all at once", not gradually, which could mean a bigger downtime for server and clients during that time (but I'm not 100% familiar how upgrades are done in enterprise)

Probably we'll need to make this very clear in release notes and communications.
"Don't upgrade to NC 22 if you can't upgrade all the client apps also at the same time".

Just thinking out loud. I understand that there is no way to keep compatibility.

Reply:
> Also it won't be possible to first upgrade all client apps first and then the server, so the migration will have to be done "all at once", not gradually, which could mean a bigger downtime for server and clients during that time (but I'm not 100% familiar how upgrades are done in enterprise)

No in theory apps can already start rolling out, the capability will tell them if v4 is available on the server and otherwise they use the old v1-3 apis. It's just not possible anymore to serve v1-3 from a current server. but clients can be compatible with v1-4 at the same time (with some if v4 then X else Y logic)